qa/src/test/java/io/camunda/migrator/qa/runtime/SkipAndRetryProcessInstancesTest.java

```diff
@@ -86,8 +86,7 @@ public void shouldSkipProcessWithMultiInstanceServiceTask() {
     // then
     assertThatProcessInstanceCountIsEqualTo(0);
 
-    List<IdKeyDbModel> skippedProcessInstanceIds =
-        idKeyMapper.findSkippedByType(IdKeyMapper.TYPE.RUNTIME_PROCESS_INSTANCE, 0 ,10).stream().toList();
+    List<IdKeyDbModel> skippedProcessInstanceIds = dbClient.findSkippedProcessInstances().stream().toList();
```

Comment:
💭  This change is also in main as well, does not seem to cause a merge conflict.

Reply:
Just wanted to have a green CI so I cherry-picked your change before it was on main. GitHub can handle this quite well.